### ISSUE TITLE
false positives on sites behind cloudflare

### ISSUE THREAD

**Issue by soredake** [cadeyrn/bookmarks-organizer#211]
**Describe the bug**
Many sites that are behind clouldflare are always shown as "broken".

**To Reproduce**
1. Add affected site/url to bookmarks.
2. Check bookmarks with this addon.
3. Affected site/url is marked as broken.

**Expected behavior**
Sites behind cloudflare check should not be marked as broken.

**Screenshots**
![image](https://user-images.githubusercontent.com/5204968/234248878-d67bd0d5-eeaf-4789-af1a-3643adf76d46.png)

**Version information:**
- Firefox version: Firefox 112
- Bookmarks Organizer version 4.0.1

**Additional context**
Affected sites list:
```
https://ebanoe.it/
https://forum.xda-developers.com/t/xposed-app-5-0-alltrans-completely-translate-apps.3539878/
https://pcpartpicker.com/
https://gamegauntlets.com/
https://www.elevenforum.com/t/enable-or-disable-snipping-tool-auto-save-screenshots-in-windows-11.9418/
https://www.pugetsystems.com/blog/2021/06/02/Building-a-Personal-Cloud-Gaming-Server-2153/
```

**Comment by cadeyrn** [comment]
I can not reproduce. I tested all your example URLs and none of these is marked as broken for me. Can you please test if you get the same result with a new Firefox profile and Bookmarks Organizer as your only add-on? This is to make sure that no non-default browser setting or other add-ons causes conflicts.

**Comment by soredake** [comment]
@cadeyrn yes, you're right, in fresh profile i can't reproduce it either, i will try to disable addons one-by-one to determine which of them is causing it, same with settings.

**Comment by soredake** [comment]
Turns out it's because i have too many bookmarks and checking so many cloudflare sites will trigger some temporary block, closing for now
